ui/perfherder/RevisionInformation.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular/index.es2015';
+import { ListGroup, ListGroupItem } from 'reactstrap';
+
+import perf from '../js/perf';
+
+import { getRevisionUrl } from './helpers';
+
+function getRevisionSpecificDetails(
+  revision,
+  project,
+  isBaseline,
+  resultSet,
+  selectedTimeRange = null,
+) {
+  const revisionUrl = getRevisionUrl(revision, project.name);
+  const truncatedRevision = revision.substring(0, 12);
+  let baselineOrNew = isBaseline ? 'Base' : 'New';
+  if (selectedTimeRange) baselineOrNew = 'Base';
+
+  return (
+    <React.Fragment>
+      <strong>{baselineOrNew}</strong> -&nbsp;
+      {revisionUrl ? (
```

Comment:
The `revisionUrl` will always be truthy since `getRevisionUrl` is either returning a hyperlink or an empty string. You could get rid of `getRevisionUrl` entirely and move that logic into here, only generating the url if revision is true.